recipes/rmlui/3.3/conanfile.py

```diff
@@ -0,0 +1,132 @@
+from conans import ConanFile, CMake, tools
+from six import StringIO
+import os
+
+
+class RmluiConan(ConanFile):
+    name = "rmlui"
+    description = "RmlUi - The HTML/CSS User Interface Library Evolved"
+    homepage = "https://github.com/mikke89/RmlUi"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    topics = ("conan", "css", "gui", "html", "lua", "rmlui")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "enable_rtti_and_exceptions": [True, False],
+        "font_interface": ["freetype", None],
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "with_lua_bindings": [True, False],
+        "with_thirdparty_containers": [True, False]
+    }
+    default_options = {
+        "enable_rtti_and_exceptions": True,
+        "font_interface": "freetype",
+        "fPIC": True,
+        "shared": False,
+        "with_lua_bindings": False,
+        "with_thirdparty_containers": True
+    }
+    exports_sources = ["CMakeLists.txt"]
+    generators = ["cmake", "cmake_find_package"]
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 14
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        tools.check_min_cppstd(self, self._minimum_cpp_standard)
+
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.options.font_interface == "freetype":
+            self.requires("freetype/2.10.1")
+
+        if self.options.with_lua_bindings:
+            self.requires("lua/5.3.5")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_cmake(self):
+        if not hasattr(self, "_cmake"):
+            self._cmake = CMake(self)
+
+            cmake_version_output = StringIO()
```

Comment:
Person the output of cmake is brittle, and not needed, I think.

The parsing also breaks when conan is configure to output the commands.
Check CONAN_PRINT_RUN_COMMANDS

Reply:
I'll implement @prince-chrismc's recommendation to make all of this unnecessary. :-)

Reply:
We love less code 🤣